src/constants.rs

```diff
@@ -0,0 +1,13 @@
+pub const RPC: &str = "beta-5.fuel.network";
+
+pub const TOKEN_CONTRACT_ID: &str =
```

Comment:
What token is this?

Reply:
@chlenc

Reply:
It’s our contract based on fuel token standards, not sure it need to be here

Reply:
https://github.com/compolabs/src-20